_notes/why_must_i_learn_this.md

```diff
@@ -5,148 +5,108 @@ title: Why Must I Learn This?
 
 I believe that the “why” is largely missing in today’s education system.
-
 Without it, the beauty of learning is lost.
-
-We leave the system less curious than before.
+We leave our schools less curious than when we started.
 
 That’s why I believe that the future of education needs to better address the “why”.
 
+
 # Two parters
 
 Understanding “why” has 2 parts.
+Past context and present context.
 
-First, a student need to understand the context in which that knowledge was formed.
-
+First, a student needs to understand the CONTEXT in which a concept was formed in the PAST.
 For example, let’s look at housing policies concepts.
-
 What problems were the policy makers trying to solve?
-
 What tools did they have at hand?
-
 What were their hidden incentives?
+Half the battle is won just by understanding why the concept existed to begin with.
 
-Half the battle is won just by understanding the context which the knowledge was formed.
-
-Second, the students need to understand how this relates to their world today.
-
-Application is possible if they draw similarities between the context of the past and the context of today.
-
-Only through applying the knowledge in a meaningful way is mastery of the knowledge truly achieved.
+Second, the students need to understand how that CONTEXT relates to their PRESENT world.
+Drawing similarities between context of the past and the context of today.
+Adapting and moulding the concept to apply it for themselves.
+Through this practice, we can see a true mark of mastery, and a clear demonstration of a complete understanding of “WHY I learned this”.
 
 # Poorly positioned
 
 Achieving the first part is not easy.
+Teachers that do that well are hard to come by.
 
-Teachers that can do that well are hard to come by.
-
-Achieving the second part appears to be even more challenging for teachers.
+Achieving the second part appears to be even more challenging.
 
 Why?
-
-The teachers themselves are not positioned to do the second part themselves!
-
-They do not spend extended periods of time applying the knowledge to solve real world problems.
-
-They do not have the chance to wrestle with all the fine nuances of how to manipulate that piece of knowledge.
-
-Therefore the appreciation for that knowledge is often not deep enough.
-
-The job of the school teacher is not to be the practitioner of the craft.
+Teachers are not positioned to do it well!
+They are not blessed with the opportunity to put these ideas into practice and see it through to execution.
+To wrestle with all aspects of the idea as it clashes head on with the messy nature of reality,
+and developing deep appreciation for the idea through that experience.
+That’s the job of a practitioner of the craft, not a school teacher.
 
 We know this.
+Business teachers invite successful founders to the classroom.
+Physics teachers invite astronauts to the classroom.
+Basketball coaches invite NBA players to the classroom.
```

Comment:
Maybe better to structure it as teachers already know CONTEXT is valuable

Business teachers spend classhours going through case studies of failing and rising businesses

Science teachers love to bring in engineers or scientists that engage with the same essential tools found in the lab but in far more compelling ways 


Coaches spend time looking through the replays of the very best at the game

Reply:
I removed the point about "best". Added in the idea of teachers VALUING the context which indus professionals bring.

Reply:
I think contextualising it to something all teachers do is better than what some teachers have the opportunity to do how many teachers do you know have invited nba players or big business owners into the classroom